src/components/Comment/index.jsx

```diff
@@ -0,0 +1,3 @@
+import { Comment } from './Comment';
+
+export { Comment };
```

Comment:
можна застосувати скорочений запис

```suggestion
export * from './Comment';
```

Reply:
І у всіх індексах також